addons/binding/org.openhab.binding.meteostick/README.md

```diff
@@ -19,6 +19,15 @@ The Meteostick things need to be manually added - there is no discovery in the M
 First add and configure the Meteostick bridge - the port and frequency band for your region need to be set.
 Next add the sensor and configure the channel number.
 
+### Thing Configuration
```

Comment:
"This should be mainly about its mandatory and optional configuration parameters"
-> provide a short table that describes the port and mode config parameters.